charts/elasticsearch/templates/_helpers.tpl

```diff
@@ -153,3 +153,12 @@ imagePullSecrets:
 {{ . }},
 {{- end -}}
 {{- end -}}
+
+
+{{- define "curator.indexPattern" -}}
+{{ if and .Values.global.loggingSidecar.enabled  .Values.global.loggingSidecar.indexPattern }}
+{{- .Values.global.loggingSidecar.indexPattern | squote }}
+{{ else }}
+{{- .Values.curator.age.timestring | squote}}
```

Comment:
Why do we need "squote" here? We'll definitely need to make test cases with various quoting possibilities. https://helm.sh/docs/chart_template_guide/function_list/#printf may be needed.